### DISCUSSION TITLE
Recording video

### DISCUSSION THREAD

**Discussion by jonathangiber** [getgauge/gauge#2184] [General]
Hi guys,
Im not using taiko, only gauge. Is there any easy solution to integrate with gauge that will record specs? 

Thanks!

**Comment by lirany1** [comment]
I am using 
https://opensource.zalando.com/zalenium/

**Reply by jonathangiber** [reply]
How do you integrate it with gauge?

**Reply by lirany1** [reply]
> How do you integrate it with gauge?

Actually the Selenium Grid use Zalenium so the integration is Selenium-Gauge
you can check here for example:
https://github.com/LambdaTest/gauge-selenium-sample
